peer/test_utils.go

```diff
@@ -53,14 +53,19 @@ var (
 // the channels set up.
 var noUpdate = func(a, b *channeldb.OpenChannel) {}
 
+type ctpReturnParams struct {
```

Comment:
please rename this to something else. `ReturnParams` is 12 characters of our 80 char column limit and it tells is nothing whatsoever. I'd recommend something like `TestPeerEnv`, `TestPeerSetup` or something to that effect.